devicetypes/smartthings/zigbee-button.src/zigbee-button.groovy

```diff
@@ -72,7 +72,7 @@ def parse(String description) {
     else {
         if ((description?.startsWith("catchall:")) || (description?.startsWith("read attr -"))) {
             def descMap = zigbee.parseDescriptionAsMap(description)
-            if (descMap.clusterInt == 0x0001 && descMap.attrInt == 0x0020) {
+            if (descMap.clusterInt == 0x0001 && descMap.attrInt == 0x0020 && descMap.value != null) {
```

Comment:
Do you know when this is `null`?

Reply:
Maybe for the reporting config response?